### ISSUE TITLE
streaming_delta sometimes reports a wrong delta amount

### ISSUE THREAD

**Issue by frost242** [OPMDG/check_pgactivity#166]
Hello,

@Krysztophe noticed a strange behavior of service streaming_delta. When the streaming connection is in state 'startup', the reported LSN is 0/0. In this case, check_pga uses 0/0 as the standby's current position and leads to very huge and wrong delta calculation.
We should ignore the startup state when querying the pg_stat_replication view.

For example : 
```
  pid  | usesysid |  usename   | application_name | client_addr | client_hostname | client_port |        backend_start         | backend_xmin |  state  | sent_location | write_loc
ation | flush_location | replay_location | sync_priority | sync_state  
-------+----------+------------+------------------+-------------+-----------------+-------------+------------------------------+--------------+---------+---------------+----------
------+----------------+-----------------+---------------+------------ 
 22913 |    16388 | replicator | walreceiver      | ... |                 |       39885 | 2018-02-21 16:59:00.09879+01 |              | startup | 0/0           |          
      |                |                 |             0 | async 
(1 ligne)
```

**Comment by ioguix** [comment]
We should probably pay attention to the status we return depending on the standby state.

I suppose we should return the same status than a missing standby when its state is in `startup`. What about the `catchup` state? Should we return `OK` as the standby is actually connected and streaming, trying to catchup with the primary?

**Comment by frost242** [comment]
I agree with this. There are several states that we should take account: https://www.postgresql.org/docs/10/static/monitoring-stats.html#PG-STAT-REPLICATION-VIEW
I still have to think further about the catchup state. But as the probe measures the delta of the replication stream, we probably should return UNKNOWN in this case, and probably for all states except streaming.

**Comment by ioguix** [comment]
In fact, during the catchup state, the standby is actually replicating. So we can just go through the normal code path, depending on the lag, it will raise an alert or not...

**Comment by frost242** [comment]
I don't remember if we can calculate the lag at this point. I will check when I have some time and will adapt the code accordingly.

**Comment by frost242** [comment]
Seems we can calculate the lag in catchup state. Should be fixed in PR #172

**Comment by frost242** [comment]
Patch merged
